test/commands/api/update.test.js

```diff
@@ -16,6 +16,21 @@ describe('invalid api:update command issues', () => {
     })
     .it('runs api:update with no required --file flag')
 
+  test
+    .command(['api:update', 'org/api', '-f=test/resources/valid_api.json', '--visibility'])
+    .catch(err => {
+      expect(err.message).to.equal('Flag --visibility expects a value')
+    })
+    .it('runs api:update with no required --visibility flag')
+
```

Comment:
Also verify that the command is rejected if it does not have the value of `private` or `public`.